framework/include/base/MooseError.h

```diff
@@ -223,24 +223,26 @@ mooseDeprecatedStream(S & oss, bool expired, Args &&... args)
   if (Moose::_deprecated_is_error)
     mooseError("\n\nDeprecated code:\n", std::forward<Args>(args)...);
 
-  mooseDoOnce(std::ostringstream ss; mooseStreamAll(ss, args...);
-              std::string msg = mooseMsgFmt(
-                  ss.str(),
+  std::ostringstream ss;
+  mooseStreamAll(ss, args...);
+  std::string msg =
+      mooseMsgFmt(ss.str(),
                   "*** Warning, This code is deprecated and will be removed in future versions:",
```

Comment:
could we moose do once ONLY that line?
That way we have all the deprecated messages, and this shared message only once

Reply:
Which message do you mean, the `warning this code is deprecated` one?

Reply:
Yes